bindings/wasm/examples/src/tests/4_delete_did.ts

```diff
@@ -1,4 +1,4 @@
-import {deactivateIdentity} from "../ex3_deactivate_did";
+import { deactivateIdentity } from "../0_basic/3_deactivate_did";
 
 // Only verifies that no uncaught exceptions are thrown, including syntax errors etc.
 describe("Test node examples", function () {
```

Comment:
The tests for `3_deactivate_did` and `4_delete_did` are switched. They're just in the wrong test filenames.